libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -52,10 +52,14 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
 
     if (query.seen != null) {
       requestQuery.seen = query.seen;
+    } else {
+      requestQuery.seen = { $in: [true, false] };
```

Comment:
Added so we always use the same index no matter if there is seen or read or not (with the createAt key)